### PR TITLE
Update print function to write to stderr

### PR DESCRIPTION
**Pull request by avrittrohwer**
Fixes https://github.com/civiform/civiform/issues/3967.

### CODE REVIEW

**Review by bion — CHANGES_REQUESTED** [review]
Please resolving failing CI checks

**Comment by avrittrohwer** [comment]
Fixed checks

File: cloud/shared/bin/lib/print.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import builtins
+import sys
+
+
+def print(*args, **kwargs):
```
Comment (bion) [inline cloud/shared/bin/lib/print.py:5]: suggestion (blocking): rename to `print_error` so it's clear this goes to stderr, we should be able to print info output to stdout

**Review by bion — CHANGES_REQUESTED** [review]

**Comment by avrittrohwer** [comment]
The reason I shadowed print is that this diff for this PR is way smaller.
I'm not sure what the deploy tool would use stdout for, I can't imagine
piping the deploy tool to anything other than logging.  In the future if we
need to print to stdout, we can change the shadowed print function to not
override the file named parameter and callers could pass file=sys.stdout.
Thoughts?

File: cloud/shared/bin/lib/print.py (hunk 1 above)
Comment (avrittrohwer) [reply cloud/shared/bin/lib/print.py:5]: The reason I shadowed print is that this diff for this PR is way smaller.
I'm not sure what the deploy tool would use stdout for, I can't imagine
piping the deploy tool to anything other than logging.  In the future if we
need to print to stdout, we can change the shadowed print function to not
override the file named parameter and callers could pass file=sys.stdout.
Thoughts?

Comment (bion) [reply cloud/shared/bin/lib/print.py:5]: Sorry for the slow response!

Printing error messages to stdout is fine I think. Some people may have their terminals configured to print stderr in a different color or something but it's not a big deal.

We are however already printing to stderr in a few other places, so it'd be good to get consistent.

**Review by bion — APPROVED** [review]

**Pull request merged**